From 7202351f82697455cf00543b871e9b18e8099184 Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Thu, 13 Feb 2014 13:46:59 -0800
Subject: [PATCH 450/974] netdev/phy: Clean up structure names in vsc848x.c

Cut-and-paste snafu left some bad names, no functional change.

Signed-off-by: David Daney <david.daney@cavium.com>
[Original patch taken from Cavium SDK 3.1.2-568]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/net/phy/vsc848x.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/phy/vsc848x.c b/drivers/net/phy/vsc848x.c
index 67fe374..382c061 100644
--- a/drivers/net/phy/vsc848x.c
+++ b/drivers/net/phy/vsc848x.c
@@ -738,7 +738,7 @@ static struct of_device_id vsc848x_nexus_match[] = {
 };
 MODULE_DEVICE_TABLE(of, vsc848x_nexus_match);
 
-static struct platform_driver mdio_mux_gpio_driver = {
+static struct platform_driver vsc848x_nexus_driver = {
 	.driver = {
 		.name		= "vsc848x-nexus",
 		.owner		= THIS_MODULE,
@@ -752,7 +752,7 @@ static int __init vsc848x_mod_init(void)
 {
 	int rv;
 
-	rv = platform_driver_register(&mdio_mux_gpio_driver);
+	rv = platform_driver_register(&vsc848x_nexus_driver);
 	if (rv)
 		return rv;
 
@@ -765,7 +765,7 @@ module_init(vsc848x_mod_init);
 static void __exit vsc848x_mod_exit(void)
 {
 	phy_driver_unregister(&vsc848x_phy_driver);
-	platform_driver_unregister(&mdio_mux_gpio_driver);
+	platform_driver_unregister(&vsc848x_nexus_driver);
 }
 module_exit(vsc848x_mod_exit);
 
-- 
2.6.2

